plan/preprocess.go

```diff
@@ -139,13 +139,12 @@ func isConstraintKeyTp(constraints []*ast.Constraint, colDef *ast.ColumnDef) boo
 	for _, c := range constraints {
 		// If the constraint as follows: primary key(c1, c2)
 		// we only support c1 column can be auto_increment.
-		if colDef.Name.Name.L != c.Keys[0].Column.Name.L {
-			continue
-		}
-		switch c.Tp {
-		case ast.ConstraintPrimaryKey, ast.ConstraintKey, ast.ConstraintIndex,
-			ast.ConstraintUniq, ast.ConstraintUniqIndex, ast.ConstraintUniqKey:
-			return true
+		if colDef.Name.Name.L == c.Keys[0].Column.Name.L {
```

Comment:
I don't think we need to change here, it's a different style but not obviously better.

Reply:
OK.I will revert it.